mtags/src/main/scala/scala/meta/internal/pc/CompilerInitializationOptions.scala

```diff
@@ -6,17 +6,44 @@ package scala.meta.internal.pc
  * doesn't explicity send in anything, we default everything to None to
  * signify the client didn't set this. This will then default to what is
  * currently set (more than likely the defaults) in the PresentationCompilerConfigImpl.
+ *
+ * @param completionCommand command identifier to trigger completion.
+ * @param isCompletionItemDetailEnabled whether the `CompletionItem.detail` field should be
+ *                                      populated.
+ * @param isCompletionItemDocumentationEnabled whether the `CompletionItem.documentation`
+ *                                             field should be populated.
+ * @param isCompletionItemResolve whether the client wants Metals to handle `completionItem/resolve`
+ * @param isHoverDocumentationEnabled whether to include docstrings in a `textDocument/hover`.
+ * @param isSignatureHelpDocumentationEnabled whether the `SignatureHelp.documentation` field
+ *                                            shouldbe populated.
+ * @param overrideDefFormat whether the override should include an a unicode icon or only ascii.
```

Comment:
an a -> an only?

Reply:
A yea, that's confusing. Just wanted an `a` there. Fixed.